### PR TITLE
refactor(console): refactor the logic of getting connector name in preferred language

### PR DESCRIPTION
**Pull request by darcyYe**
## Summary

Refactor the logic of getting the connector's name of the preferred language.

## Testing

N/A

### CODE REVIEW

**Review by wangsijie — APPROVED** [review]

File: packages/console/src/pages/Connectors/components/Guide/index.tsx [hunk 1]
```diff
@@ -31,10 +32,10 @@ const Guide = ({ connector, onClose }: Props) => {
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
   const { id: connectorId, type: connectorType, name, configTemplate, readme } = connector;
 
-  const locale = i18next.language;
+  const localeRaw = i18next.language;
+  const locale = languageEnumGuard.parse(localeRaw);
```
Comment (wangsijie) [inline packages/console/src/pages/Connectors/components/Guide/index.tsx:36]: will fallback to?

Comment (darcyYe) [reply packages/console/src/pages/Connectors/components/Guide/index.tsx:36]: we set 'en' as fallbackLng when initializing i18next, so I think i18next.language will be 'en' if not specified?

Comment (wangsijie) [reply packages/console/src/pages/Connectors/components/Guide/index.tsx:36]: `parse` will throw an error?

Comment (darcyYe) [reply packages/console/src/pages/Connectors/components/Guide/index.tsx:36]: I thought that if `localRaw` is not in the `name`'s keys, it is informative enough to throw ZodError to show that `localRaw` is currently not supported?
Or should we throw an error in some other error types?

Comment (wangsijie) [reply packages/console/src/pages/Connectors/components/Guide/index.tsx:36]: I think this should fallback to `en`, instead of killing the app.

Comment (wangsijie) [reply packages/console/src/pages/Connectors/components/Guide/index.tsx:36]: Besides, this is a refactor PR, is not expected to change the logic.

Comment (darcyYe) [reply packages/console/src/pages/Connectors/components/Guide/index.tsx:36]: sounds reasonable, will fix this soon.

Comment (darcyYe) [reply packages/console/src/pages/Connectors/components/Guide/index.tsx]: Updated, PTAL

**Pull request merged**